packages/lib/shim-init-node.js

```diff
@@ -551,6 +555,10 @@ function shimInit(options = null) {
 						});
 					});
 
+					request.on('timeout', () => {
+						request.destroy(new Error('Request timed out.'));
```

Comment:
Do we know the timeout duration in this context? And if so could you add it to the error message?

Reply:
I think so, since it is passed in requestOptions to create the request object. I'm going to add it.

Reply:
Error thrown by follow-redirects library when there is too many redirects:

```
2023-11-10 12:11:16: [warn] routes/notes: Cannot download image at https://ctRz004.na1.hubspotlinks.com/Cto/2J+113/ctRz004/VX1V1L33NcNZVzDZZD7trZvDW824bMN5578pqW8Lz_VM8fYH4X29l1 CustomError {code: 'ERR_FR_TOO_MANY_REDIRECTS', message: 'Maximum number of redirects exceeded', stack: 'Error [ERR_FR_TOO_MANY_REDIRECTS]: Maximum nu…dChunk (node:internal/streams/readable:297:9)'}
```